app/Http/Controllers/BuildController.php

```diff
@@ -548,15 +548,19 @@ public function viewUpdatePageContent(): JsonResponse
         $next_buildid = $this->build->GetNextBuildId();
 
         if ($previous_buildid > 0) {
-            $menu_response['previous'] = url("/build/$previous_buildid/update");
+            $menu_response['previous'] = "/build/$previous_buildid/update";
         } else {
             $menu_response['previous'] = false;
         }
 
-        $menu_response['current'] = url("/build/$current_buildid/update");
+        if ($current_buildid > 0) {
+            $menu_response['current'] = "/build/$current_buildid/update";
+        } else {
+            $menu_response['current'] = false;
+        }
```

Comment:
It should never be possible for there to be a valid build update page (`/build/<id>/update`) without at least one build existing.  Thus, unless we have another error somewhere else, there should be no way for the latest build to be unavailable.

Reply:
I thought so too, but I found instances where that does happen. For example, if you navigate to `/build/16` (the build summary page) in the development instance, scroll to the "previous-this-next build" table section and click on the cell value in the `Update` row, you could get to `/build/16/update`, where there is no `PREV` or `NEXT`, and the build id in the link of the `CURR` button evaluates to 0. I'm not sure I understand the logic of what should be on the page, so for now this PR at least fixes the link. Perhaps this could be addressed in a future PR.

Reply:
Seems reasonable to me.  If there's a way to get to a state where there is no "current" update, we should definitely have a check for that.